style.css

```diff
@@ -0,0 +1,619 @@
+  background-image: url("images/contact\ form\ background\ shapes@2x.svg");
+  background-color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  width: 50%;
+  margin-left: 25%;
+}
+
+.dubbed {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  margin-top: 103px;
+}
+
+#clickin {
+  border: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 109%;
+  margin: 20px 0;
+  font-size: 20px;
+  padding: 10px;
+  color: #172b4d;
+  line-height: 20px;
+  border-radius: 10px;
+}
+
+#clickin:focus {
+  border: 1px solid #172b4d;
+}
+
+#textbox {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-size: normal;
+  line-height: 20px;
+  width: 109%;
+  border-radius: 8px;
+  border: 1px solid #fff;
+  min-height: 140px;
+  padding: 10px;
+  margin: 10px 0;
+}
+
+.button_sent {
+  border-radius: 8px;
+  margin: 10px 0;
+  align-self: center;
+  border: 1px solid #fff;
+  color: #6070ff;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 12px 16px;
+}
+
+.button_sent:hover {
+  color: #fff;
+  background: #6070ff;
+}
+
+.details {
+  padding: 8px;
+  margin: 0 auto;
+}
+
+.centerbutton {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 80px;
+}
+
+#wrapper {
+  max-width: 1050px;
+  margin: 60px auto 60px auto;
+  background-color: #fff;
+}
+
+#inner {
+  display: table;
+  width: 100%;
+  table-layout: fixed;
+}
+
+.about_myself {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  margin-top: 60px;
+  margin-bottom: 20px;
+  font-weight: 700;
+  color: #172b4d;
+  line-height: 52px;
+  position: relative;
+}
+
+@media screen and (min-width: 768px) {
+  .beginning {
+    padding: 0 50px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navigation_option {
+    display: flex;
+    align-self: center;
+    justify-self: center;
+    flex-direction: row;
+  }
+
+  .toolbar {
+    min-height: 916px;
+    background-repeat: no-repeat;
+    display: flex;
+    justify-content: center;
+    align-items: flex;
+    margin-left: 100px;
+  }
+
+  .mid {
+    align-self: center;
+    justify-self: center;
+    max-width: 920px;
+  }
+
+  .start {
+    margin: 25px 100px;
+  }
+
+  .topic {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .topic:nth-child(even) .summary {
+    order: 2;
+  }
+
+  .topic2 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .topic2:nth-child(even) .summary {
+    order: 2;
+  }
+
+  .topic3 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .topic3:nth-child(even) .summary {
+    order: 2;
+  }
+
+  .topic4 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .topic4:nth-child(even) .summary {
+    order: 2;
+  }
+
+  .instruction {
+    margin-left: 15px;
+  }
+
+  .information {
+    grid-template-columns: 1fr 1fr;
+    padding: 25px 50px;
+  }

```

Comment:
For this section, loads of aspects are not respecting the Figma design
![edited](https://user-images.githubusercontent.com/83596135/203148762-bac2f26c-6570-4e20-9373-d5e71aa42740.png)

- [ ] Kindly, remove the white space between the text from the left side and the technologies from the right side of the page;
- [ ] Please remove the white space between the cards with technologies;
- [ ] I would advise centring inside each box the technology icon and the name of the technology.
- [ ] The arrows are not displayed; please double check why (maybe is the path again?);